src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -61,6 +65,36 @@ private boolean isFileContentEqual(
     }
   }
 
+  @Test
+  public void testClientInfoConnectionProperty() throws Throwable {
+    Connection connection = null;
+    Statement statement = null;
+    ResultSet res = null;
+
+    try {
+      Properties props = new Properties();
```

Comment:
In the unit test, can you test below json?  This is an example what spark connector use.
 {
 "spark.version" : "3.0.0",
 "spark.snowflakedb.version" : "2.8.5",
 "spark.app.name" : "SnowflakeSourceSuite",
 "scala.version" : "2.12.11",
 "java.version" : "1.8.0_221",
 "snowflakedb.jdbc.version" : "3.13.2"
}